docs/index.md

```diff
@@ -1,13 +1,25 @@
+---
+hide:
+  - navigation
+---
+
 Gaffer 2
 ========
 
-*These Docs are a work in progress for Gaffer v2, [the docs for Gaffer v1 can be found here](../v1docs/)*.
+!!! info "Under construction"
+
+    This documentation is under construction and focusses on Gaffer version 2.0.0 and above.
+    [The previous docs for Gaffer 1.x can be found here](../v1docs/).
 
-For information and migration steps for Deprecated classes which have been removed in Gaffer 2 alpha 1, [please see the deprecations page](gaffer2.0/deprecations.md).
+Gaffer 2.0.0 was released in May 2023. The final release of Gaffer 1.x was [1.23.0](https://github.com/gchq/Gaffer/releases/tag/gaffer2-1.23.0), there are no further releases to the 1.x branch planned.
```

Comment:
Maybe we should link the 2.0 release too?

Reply:
I think I intended to and forgot. It's a shame it doesn't list all the changes since 1.23 in the GitHub generated change log.